src/features/swap/hooks/useStaking.ts

```diff
@@ -28,6 +28,19 @@ export function useStaking() {
 
   const client = usePublicClient();
 
+  const _estimateGas = useCallback(
```

Comment:
because this seems to get passed around and its not a generic estimate gas but estimate for manager contract, maybe add estimateManagerGas ?

Reply:
Done